numpy/testing/decorators.py

```diff
@@ -9,7 +9,8 @@
 
 # 2018-04-04, numpy 1.15.0
 warnings.warn("Importing from numpy.testing.decorators is deprecated "
-              "since numpy 1.15.0, import from numpy.testing instead.",
+              "since numpy 1.15.0, import from numpy.testing or "
+              "use the namespace numpy.testing.dec instead.",
```

Comment:
I think that is confusing because the `numpy.testing.dec` module does not exist. One could do
```
import numpy.testing
numpy.testing.dec.knownfailif
#etc.
```
or more simply
```
from numpy.testing import dec
dec.knownfailif
#etc.
```

Reply:
Yeah, I agree it's confusing. I was trying to get around this by saying "namespace" instead of "module", but they clearly weren't different enough to make the distinction usefully clear.

The better fixes I can think of involve changing the wording to something like:

> The module numpy.testing.decorators is deprecated
> since numpy 1.15.0 and will be removed, import numpy.testing or numpy.testing.dec
> and use those namespaces instead

But the larger the change, the greater probability of breaking pytest `ignore` filters people already have set up, so I think I'll close and let people's search skills (or module inspection skills) take them to the solution instead.